src/Compilers/Core/Portable/PEWriter/PeWriter.cs

```diff
@@ -1459,7 +1459,7 @@ private void WriteDebugTable(Stream peStream, MetadataSizes metadataSizes, out l
             // characteristics:
             writer.WriteUint(0);
 
-            // timestamp from NT headers
+            // PDB stamp
             timestampOffset = writer.BaseStream.Position + peStream.Position;
             writer.WriteUint(pdbStamp);
 
```

Comment:
Above, when we get "age" from GetDebugDirectoryGuidAndStampAndAge, should we assert that "age == 1" (with a comment)?